e2e/_suites/fleet/ingest-manager_test.go

```diff
@@ -36,9 +36,15 @@ const ElasticAgentServiceName = "elastic-agent"
 // FleetProfileName the name of the profile to run the runtime, backend services
 const FleetProfileName = "fleet"
 
+var agentVersionBase = "8.0.0-SNAPSHOT"
+
+// agentVersion is the version of the agent to use
+// It can be overriden by ELASTIC_AGENT_VERSION env var
+var agentVersion = agentVersionBase
+
 // stackVersion is the version of the stack to use
 // It can be overriden by STACK_VERSION env var
-var stackVersion = "8.0.0-SNAPSHOT"
+var stackVersion string
```

Comment:
This won't work, if no value, since  stackVersion  = `env. STACK_VERSION`

Should it be?

```suggestion
var stackVersion = agentVersionBase
```

Reply:
I've just sent a commit on top: https://github.com/elastic/e2e-testing/pull/426/commits/e74b68b50d508696c0e7b615e7f97e0ff0f773a9#diff-4e9ca5d5986bb705c62581784ed63010d9ad3b449c13a7164930a49b77965c77R74

Reply:
You are right. I sent a commit on top simplifying the initialisation, thanks!